### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by michmike**
updating the contribution guide for API reviews
/sig windows

### CODE REVIEW

**Comment by michmike** [comment]
cc: @ddebroy  for review and approval

File: sig-windows/CONTRIBUTING.md [hunk 1]
```diff
@@ -75,6 +76,10 @@ The detailed steps here are based off an example PR on GitHub: [https://github.c
 6. Deploy your own cluster, including Windows Nodes
 7. Test Your Changes
 
+## API Considerations
+
+If you modifying an API in the SIG-Windows codebase, make sure you are aware of the API [guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md) and [conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md) used in Kubernetes. This [document offers guidelines for API reviewers that API developers should always have in consideration](https://docs.google.com/document/d/1pkCYjr_OLCRUk2e606eMa2PEIwJdFHAfU5slXtBcKj8/edit).
```
Comment (nikhita) [inline sig-windows/CONTRIBUTING.md]: nit: Can you update the links to be relative? The first link would become `/contributors/devel/sig-architecture/api_changes.md` and so on...

Comment (nikhita) [reply sig-windows/CONTRIBUTING.md]: one more tiny nit: can you wrap the lines into 80 chars? :grimacing:

**Pull request merged**